src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -43,12 +44,21 @@ public void removeTag() throws IllegalValueException, PersonNotFoundException {
         assertTrue(modelManager.equals(modelManagerCopy));
 
         //Removing a tag from an address book changes it
-        ObservableList<Tag> tags = modelManager.getAddressBook().getTagList();
-        ObservableList<Tag> tagsWithoutFriends = tags.filtered(x -> !x.tagName.equals("friends"));
+        ObservableList<Tag> tags = modelManager
+                                               .getAddressBook()
```

Comment:
I think it would be readable if you leave it on one line, as it also complies with the rest of the code base.